components/dropdown/test/dropdown-content.test.js

```diff
@@ -243,24 +243,6 @@ describe('d2l-dropdown', () => {
 		});
 	});
 
-	describe('scrollTo', () => {
```

Comment:
As mentioned, `scrollTo` is only used to support a visual-diff test that captures the top scroll shadow. The `boundary` is no longer supported, and this unit test it not really needed... removing it.